src/OpenTK.Windowing.Desktop/NativeWindow.cs

```diff
@@ -565,10 +570,12 @@ public unsafe NativeWindow(NativeWindowSettings settings)
                 GLFW.WindowHint(WindowHintInt.BlueBits, modePtr->BlueBits);
                 GLFW.WindowHint(WindowHintInt.RefreshRate, modePtr->RefreshRate);
                 WindowPtr = GLFW.CreateWindow(modePtr->Width, modePtr->Height, _title, monitor, null);
+                Hwnd = GLFW.GlfwGetWin32Window(WindowPtr);
             }
             else
             {
                 WindowPtr = GLFW.CreateWindow(settings.Size.X, settings.Size.Y, _title, null, null);
+                Hwnd = GLFW.GlfwGetWin32Window(WindowPtr);
```

Comment:
Since the call is the same in both branches of the `if` statement, should we put it outside the statement?